### ISSUE TITLE
Helix runs on linux x64 are timing out

### ISSUE THREAD

**Issue by davidfowl** [dotnet/corefx#35220]
My last couple of helix runs are timing out https://dev.azure.com/dnceng/public/_build/results?buildId=91865&view=logs. Taking hours to run instead of minutes.

cc @ViktorHofer

**Comment by stephentoub** [comment]
I asked @safern about this yesterday and he said he was investigating.

**Comment by safern** [comment]
It looks like the queues that are pretty full or not running any jobs are Ubuntu16.04 and 18.04. 

I've opened: https://github.com/dotnet/core-eng/issues/5209 to have engineering team take a look at why are the queues down. Either we're sending too much jobs (because we're sending from jenkins and azure devops) + we enabled corefx official build to be per commit (one at a time, with batched commits). So on Friday we had 8 official builds, this was with the intention on accelerating the process for corefx to flow to other repos. 

So, if that is the case we would need to get more machines to keep it that way.

In the meantime, I'm going to submit a PR to remove those queues, and also I will start disabling jenkins job to avoid clogging the queues.
